google-http-client/src/main/java/com/google/api/client/http/GZipEncoding.java

```diff
@@ -43,6 +43,7 @@ public void close() throws IOException {
             try {
               flush();
             } catch (IOException ignored) {
+                // Nothing else we can do here
```

Comment:
I'm not sure this is true. I wonder why we don't propagate the IOException in this case?

Reply:
I have no clue.

Reply:
```suggestion
                // fall through
```